en/contribute/language.md

```diff
@@ -0,0 +1,12 @@
+Language
+========
+
+The development language is English. All comments and documentation in common repositories should be written in English, so that contributions can be made by developers around the world.
+
+
+Country-specific repositories
+-----------------------------
+
+Some repositories define the social and fiscal system of a specific country. In such cases, the language used throughout issues and pull requests should be one of the native ones of that country.
```

Comment:
Vocabulary: in English we must use "tax and benefit system", not "social and fiscal system".

Note that "tax benefit system" is to avoid too, because it means a system in which you make benefits on taxes...

We could add a glossary/lexicon page to refer to, it's a [feature of gitbook](https://toolchain.gitbook.com/lexicon.html).